app/cli/cmd/root.go

```diff
@@ -269,12 +269,14 @@ func parseToken(token string) (*ParsedToken, error) {
 
 		switch aud[0].(string) {
 		case apiTokenAudience:
-			return &ParsedToken{Type: "api-token"}, nil
+			tokenID := claims["jti"].(string)
+			return &ParsedToken{Type: "api-token", ID: tokenID}, nil
 		case userAudience:
 			userID := claims["user_id"].(string)
 			return &ParsedToken{Type: "user", ID: userID}, nil
 		case robotAccountAudience:
-			return &ParsedToken{Type: "robot-account"}, nil
+			tokenID := claims["jti"].(string)
+			return &ParsedToken{Type: "robot-account", ID: tokenID}, nil
```

Comment:
btw, organization has been added and will be available in both robot-account and api-token once #795 is merged :)

Reply:
Awesome